src/main/java/seedu/momentum/logic/commands/EditCommand.java

```diff
@@ -46,7 +51,7 @@
     private final EditProjectDescriptor editProjectDescriptor;
 
     /**
-     * @param index of the project in the filtered project list to edit
+     * @param index                 of the project in the filtered project list to edit
```

Comment:
Should there be a big space here?

Reply:
It's just the work of reformat code to align with the next @param